src/business.ts

```diff
@@ -168,4 +173,36 @@ export class BusinessApi {
     }
     return { success: !resp?.payload?.errorCode, body: { ...resp.payload } }
   }
+
+  /*
+   * Function to take a Business token id, some Business attributes, and update
+   * the Business with that Id in Marqeta with these values. The return value will be
+   * the updated Business.
+   */
+  async update(businessTokenId: string, business: Partial<Business>): Promise<{
```

Comment:
If the `businessTokenId` is really the `id` in the `Business`, then why not just make this signature:

```typescript
  async update(business: Partial<Business>): Promise<{
```

and then just inside the function you could check the value of `business?.id` like we do in so many other functions, and return an error with a message if it's not there.

If the `token` is not the `id`, then it has to be in there... right? And from there you get the idea... Right?